splunktaucclib/rest_handler/handler.py

```diff
@@ -102,13 +102,39 @@ def check_existing(self, name):
             else:
                 return None
 
+        def basic_name_validation(name):
```

Comment:
Add type annotations

Reply:
it's self.callerArgs.id and based on core its defined as etree.subelement.text so I set it as str